packaging/python/build_wheels.bash

```diff
@@ -131,11 +131,27 @@ set_cibw_environment() {
 build_wheel_portable() {
     platform="${1}"
     echo "[BUILD WHEEL] Building with cibuildwheel for ${platform}"
-    local skip=
-    setup_venv "$(command -v python3)"
-    (( skip )) && return 0
 
-    python -m pip install cibuildwheel
+    # cibuildwheel >=3 requires that the underlying Python version be at least 3.11
```

Comment:
To clarify, the version of cibuildwheel running on the _host machine_ must be 3.11 or above; this has no impact on the version of Python that cibuildwheel uses to build the wheel itself (e.g. we can use 3.11 on the host, and still build a wheel for 3.9 or 3.13).